test/e2e/service_to_service_test.go

```diff
@@ -88,6 +89,9 @@ func testProxyToHelloworld(t *testing.T, clients *test.Clients, helloworldDomain
 	defer test.TearDown(clients, names)
 	resources, err := v1a1test.CreateRunLatestServiceReady(t, clients, &names, &v1a1test.Options{
 		EnvVars: envVars,
+		RevisionTemplateAnnotations: map[string]string{
+			autoscaling.WindowAnnotationKey: "7s",
```

Comment:
I think I understand why the call from zero cases need this, but it is less clear to me the benefit in setting this here and on line 165.

Reply:
For uniformity.